### ISSUE TITLE
Silent Hypoxemia

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7108]
**Preferred term label:**
Silent Hypoxemia

**Synonyms**
- **related synonym:** silent hypoxia
- **laymans term:** happy hypoxia

**Definition (free text, please give PubMed ID)**
Abnormally low blood oxygen level without the presence of dyspnea/dyspnoea. [PMID: 32539537]

**Parent term (use hpo.jax.org/app)**
HP:0012418 - Hypoxemia

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Database cross reference**
SNOWMEDCT_US:66331000122100 [(emerging concept - not yet released)](https://www.mscbs.gob.es/profesionales/hcdsns/areaRecursosSem/snomed-ct/SNOMED_doc/Conceptos_relacionados_SARS-CoV-2-Version11.0_20201130_en.pdf)

**Comment by pnrobinson** [comment]
adding as HP_0034038, thanks
